src/profile-logic/import/chrome.js

```diff
@@ -799,6 +775,29 @@ async function processTracingEvents(
     }
   }
 
+  if (profileUrl) {
+    // Get profile start time from profile URL if available. Timestamp format is what
+    // is generated by the Chrome DevTools UI when saving profiles. Technically, that
+    // timestamp is of profile save instead of profile start, but one can generate a
+    // more accurate start timestamp through e.g. handling the Chrome DevTools
+    // Protocol `Profiler.consoleProfileStarted` event.
```

Comment:
I don't understand how "one" can do something here. Is that something we can do now, or something that could be done in the future?

Reply:
I think that's the event that's mentioned here: https://chromedevtools.github.io/devtools-protocol/1-2/Profiler/#event-consoleProfileStarted
It's an event that's a part of CDP.

I can't find the same event inside the profile data, so I'm not so sure how to do that. Maybe we can do that for chrome extension. But I'm not so sure, maybe @kazarmy had something else in mind while writing that.

Reply:
Ah I see. We also have this one btw: https://github.com/firefox-devtools/firefox-profiler-for-chrome We recently published it on the Chrome Web Store. That's why I was spending some time on improving the importer.

Reply:
Oh cool! I do prefer to programmatically start the Chrome profiler if possible, though I understand this might not be the best way to spend precious engineer time in a work environment.